src/modules/modal/components/modal.css

```diff
@@ -0,0 +1,60 @@
+/* Modal Header */
+.modal-header {
+    padding: 2px 16px;
+    color: var(--color);
+  }
+
+  .modal-title {
+  color: var(--primary-color);
+  margin: 0;
+  text-align: center;
+}
+
+@media screen and (max-width: 600px) {
+  
+  .modal-title{
+    display: none;
+  }
+  
+}
+
+.modal-body{
+  /*background-color: var(--secondary-bg-color);*/
+  margin: 10px;
+  padding: 10px;
+}
+  
+  /* Modal Body */
+  .modal-content {    
+    background-color: var(--primary-bg-color);
+    margin: 15% auto; /* 15% from the top and centered */
+    padding: 2px;
+    border: 1px solid var(--primary-color);
+    width: 80%; /* Could be more or less, depending on screen size */
+  }
+  
+  /* Modal Footer */
+  .modal-footer {
+    padding: 2px 16px;
+    color: var(--color);    
+    display:flex;
+    justify-content: center;
+  }
+  
+  /* Modal Content */
+  .modal {
+    display:none;
+    z-index: 100;
```

Comment:
Instead of manually and static setting:
find max z-index in document and set z-index of modal to maxZIndex++
only onWindowLoad() or on every onStateChanged ?

Reply:
In my view a static z-index is enough for the moment, we can implement a more sophisticated mechanism when needed later...